core/executor/src/wasm_executor.rs

```diff
@@ -518,6 +518,14 @@ impl_function_executor!(this: FunctionExecutor<'e, E>,
 
 		Ok(0)
 	},
+	ext_submit_extrinsic(msg_data: *const u8, len: u32) => {
+		let extrinsic = this.memory.get(msg_data, len as usize)
```

Comment:
Do we maybe want to fail, when someone tries to call it `on-chain`?

Reply:
Yeah return an `Error`. I don't know what happens with this error, but we will see it :D

Reply:
Spoke with Sergei and errors are converted to traps in the wasm environment, so I've also added a panicking implementation in case of std environment.

Reply:
Spoke with Sergei and the `Error`s are converted to traps in wasm environment so I've also added a panicking implementation in `sr_io::with_std`